src/activy/register.py

```
@@ -1,12 +1,20 @@
 import subprocess
+import sys
 import time
 import shlex
 
+import cv2
```

Ruff (0.11.9)

`cv2` imported but unused

Remove unused import: `cv2`

(F401)